indico/modules/events/abstracts/models/abstracts.py

```diff
@@ -386,7 +386,22 @@ def can_access(self, user):
             return True
         if any(x.person.user == user for x in self.person_links):
             return True
-        return False
+        return self.can_review(user)
+
+    def can_review(self, user):
+        if not user:
+            return False
+        elif self.event_new.can_manage(user):
```

Comment:
Is this so? I think we agreed that only people listed as reviewer would be able to review.